webserver.py

```diff
@@ -116,6 +125,33 @@ def determine_color(uk_time):
         return STATUS_OUT
 
 
+def get_train_details():
+    if not TRANSPORT_APP_KEY or not TRANSPORT_APP_ID or not FAVOURITE_STATION:
+        return {"train_uid": ''}, ''
+    address = "http://transportapi.com/v3/uk/train/station/{}/live.json?" \
```

Comment:
can you move `uk` into a config variable at the top (something like `SHORT_COUNTRY`)? Given we let them define the country elsewhere we should let them pick it here.

Reply:
Never mind, looks like it only supports the UK rail system :,(